phpcs.xml.dist

```diff
@@ -56,11 +56,6 @@
         <exclude-pattern>src/*</exclude-pattern>
     </rule>
 
-    <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint">
```

Comment:
Response to https://github.com/doctrine/data-fixtures/pull/459#issuecomment-1991384911: This rule is applied by default. This config was to exclude `src` dir.

Reply:
Right. I missed that you've changed the config already. 🙈